MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,575 @@
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ShowLabel = false;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing layout settings
+
+You can change the label width, label position and editor width. This is possible by using the `GenerateDataFormItem` event. In the event, change the layout by using the `DefaultLayoutSettings` property in the `DataFormItem`.
+
+By default label position is left.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+                e.DataFormItem.DefaultLayoutSettings.LabelWidth = 100;
+                e.DataFormItem.DefaultLayoutSettings.EditorWidth = 200;
+            }
+                
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Background colour
+
+You can change the background colour for the label and editor by using `GenereateDataFormItem` event.
+
+You can change the background colour for the label and editor by using the `Background` property in `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Brush.Azure; 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+#### Setting order for editor
+
+By default the editor can be placed in the order of how you declare a propety in model class
+
+You can change the order of item in a row only if the column count more than 1 [If the column count = n , you can place a item upto n-1 position].
+
+You can change the order of item's within a row using `ItemsOrderInRow` property.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ColumnCount = 3;
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 1;
+                e.DataFormItem.ItemsOrderInRow = 2;  
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Label icon
+
+You can set label icon instead of using label text. Example:- You have a PhoneNumber editor, and the editor label is PhoneNumber. Here You can use a phone image for the label.
```

Comment:
remove 

Right-click on the phone image and change the build action to "Embedded resource" and configure an image in.csproj. Directly add the name of the image by using the `LeadingLabelIcon` property in `DataFormItem`.

Reply:
removed